homework_basic_components/src/task3/Task3.java

```diff
@@ -0,0 +1,70 @@
+package task3;
+
+import java.awt.GridLayout;
+import java.awt.event.KeyAdapter;
+import java.awt.event.KeyEvent;
+
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JPasswordField;
+import javax.swing.JTextField;
+
+public class Task3 extends JFrame {
+	private static final long serialVersionUID = -8404390128424131633L;
+	private String password = "1234";
+	private String username = "Mujo";
+	
+	
+	JPanel panel = new JPanel();
+	JLabel usernameLbl = new JLabel("Username:");
+	JTextField usernameTf = new JTextField();
+	JLabel passwordLbl = new JLabel("Password:");
+	JPasswordField passwordPf = new JPasswordField();
+	
+	public Task3() {
+		panel.setLayout(new GridLayout(2, 2));
+		
+		passwordPf.addKeyListener(new KeyAdapter() {
+
+			@Override
+			public void keyPressed(KeyEvent e) {
+				String pass = "";
+				if(e.getKeyCode() == KeyEvent.VK_ENTER) {
+					for (int i = 0; i < passwordPf.getPassword().length; i++) {
+						pass += passwordPf.getPassword()[i];
```

Comment:
Why not just get the content from the password field rather than doing it this way.